example/styles.css

```diff
@@ -1102,6 +1102,20 @@ a.cf-link--disabled {
 }
 
 /**
+ * Wizard
+ */
+
+.cf-wizard .cf-modal__footer {
+  display: flex;
```

Comment:
It realy need flexbox to render the buttons properly anchored to the left and right.

`cf-component-flex` won't quite do it today. It doesn't support much, just 2 spacing options. I kinda think we should add justifyContent, alignItems, and alignContent support to it, which would solve this problem and many others.